src/libraries/sendtohelix-browser.targets

```diff
@@ -51,7 +49,7 @@
     <EnableDefaultBuildHelixWorkItems>false</EnableDefaultBuildHelixWorkItems>
 
     <!-- on unix CI has emscripten provisioned in $(EMSDK_PATH) as `/usr/local/emscripten`. -->
-    <EMSDK_PATH Condition="$([MSBuild]::IsOSPlatform('WINDOWS')) and '$(EMSDK_PATH)' == ''">$(RepoRoot)src\mono\wasm\emsdk\</EMSDK_PATH>
+    <EMSDK_PATH Condition="$([MSBuild]::IsOSPlatform('WINDOWS')) and '$(EMSDK_PATH)' == ''">$(RepoRoot)src\mono\browser\emsdk\</EMSDK_PATH>
```

Comment:
Can we use `ProvisionEmscriptenDir` here?

Reply:
Good idea. We can move that property to the top level `Directory.Build.props`, and then use it here.

Reply:
Though the property put in D.B.props should be `$(InTreeEmSdk)`, and that can be used in src/tests/FunctionalTests/WebAssembly/Directory.Build.props too.